### PR TITLE
Fix Edit toolbar hides the last form item on mobile

### LINKED ISSUE #2463: can't see last items of edit form on small devices
Hi!
I can't see last item on the edit/create form on small screen devices. it is hidden with save button panel

[sample project](https://codesandbox.io/s/qk3rz6mj5w)

### PR DESCRIPTION
**Pull request by vedmalex**
making bottom padding for last child to be 70px for xs media size

Fix #2463

### CODE REVIEW

File: packages/ra-ui-materialui/src/layout/CardContentInner.js [hunk 1]
```diff
@@ -4,18 +4,22 @@ import classnames from 'classnames';
...
+  root: {
+    paddingTop: 0,
+    paddingBottom: 0,
+    '&:first-child': {
+      paddingTop: 16,
     },
-};
+    '&:last-child': {
+      paddingBottom: 16,
+      // Reset on mouse devices
```
Comment (djhi) [inline packages/ra-ui-materialui/src/layout/CardContentInner.js]: Can you remove this comment? It does not bring anything as the code is clear enough

Comment (vedmalex) [reply packages/ra-ui-materialui/src/layout/CardContentInner.js]: done

Comment (djhi) [reply packages/ra-ui-materialui/src/layout/CardContentInner.js]: I'd still like a screenshot please :)

**Comment by vedmalex** [comment]
@djhi screenshot of before anf after?

**Comment by vedmalex** [comment]
before
after

**Comment by djhi** [comment]
Thanks! I'll wait for @fzaninotto review though

**Comment by fzaninotto** [comment]
There is something wrong with your screenshot: the action buttons should not appear above the Save button. Is is a side effect of your change?

**Comment by vedmalex** [comment]
@fzaninotto this is just my custom form control from [Advanced Tutorials](https://marmelab.com/react-admin/AdvancedTutorials.html)

**Comment by vedmalex** [comment]
just see what is going on on mobile view without custom controls... it is broken
[sanbox](https://j3jxnw51yw.codesandbox.io/#/comments/create)

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**